### PR TITLE
[#747] added default language cookie + small refactor of cookie names

### PR DESCRIPTION
**Pull request by Skylsmoi**
https://github.com/tracim/tracim/issues/747
duplicate https://github.com/tracim/tracim/issues/1012

### CODE REVIEW

File: frontend/src/container/Header.jsx [hunk 1]
```diff
@@ -52,6 +57,7 @@ class Header extends React.Component {
     const { props } = this
 
     if (props.user.user_id === -1) {
+      Cookies.set(COOKIE_FRONTEND.DEFAULT_LANGUAGE, idLang, {expires: 180})
```
Comment (basilelegal) [inline frontend/src/container/Header.jsx]: the expire time 180 should be in a constant, `COOKIE_FRONTEND.DEFAULT_EXPIRE_TIME` for exemple

Comment (lebouquetin) [reply frontend/src/container/Header.jsx]: agreed

File: frontend/src/container/Header.jsx [hunk 2]
```diff
@@ -61,6 +67,7 @@ class Header extends React.Component {
     switch (fetchPutUserLang.status) {
       case 200:
         i18n.changeLanguage(idLang)
+        Cookies.set(COOKIE_FRONTEND.DEFAULT_LANGUAGE, idLang, {expires: 180})
```
Comment (lebouquetin) [inline frontend/src/container/Header.jsx]: use the new constant here also

File: frontend/src/container/Login.jsx [hunk 3]
```diff
@@ -82,8 +97,10 @@ class Login extends React.Component {
           logged: true
         }
 
-        Cookies.set('lastConnection', '1', {expires: 180})
+        Cookies.set(COOKIE_FRONTEND.LAST_CONNECTION, '1', {expires: 180})
```
Comment (lebouquetin) [inline frontend/src/container/Login.jsx]: use a constant

File: frontend/src/container/Login.jsx [hunk 4]
```diff
@@ -82,8 +97,10 @@ class Login extends React.Component {
...
         props.dispatch(setUserConnected(loggedUser))
+
+        Cookies.set(COOKIE_FRONTEND.DEFAULT_LANGUAGE, fetchPostUserLogin.json.lang, {expires: 180})
```
Comment (lebouquetin) [inline frontend/src/container/Login.jsx]: use a constant

File: frontend/src/container/Tracim.jsx [hunk 5]
```diff
@@ -91,7 +92,8 @@ class Tracim extends React.Component {
           logged: true
         }))
 
-        Cookies.set('lastConnection', '1', {expires: 180})
+        Cookies.set(COOKIE_FRONTEND.LAST_CONNECTION, '1', {expires: 180})
```
Comment (lebouquetin) [inline frontend/src/container/Tracim.jsx]: use a constant

File: frontend/src/container/Tracim.jsx [hunk 6]
```diff
@@ -91,7 +92,8 @@ class Tracim extends React.Component {
...
+        Cookies.set(COOKIE_FRONTEND.DEFAULT_LANGUAGE, fetchGetUserIsConnected.json.lang, {expires: 180})
```
Comment (lebouquetin) [inline frontend/src/container/Tracim.jsx]: use a constant

Comment (Skylsmoi) [reply frontend/src/container/Tracim.jsx]: fixed

File: frontend/src/container/Tracim.jsx (hunk 5 above)
Comment (Skylsmoi) [reply frontend/src/container/Tracim.jsx]: fixed

File: frontend/src/container/Login.jsx (hunk 4 above)
Comment (Skylsmoi) [reply frontend/src/container/Login.jsx]: fixed

File: frontend/src/container/Login.jsx (hunk 3 above)
Comment (Skylsmoi) [reply frontend/src/container/Login.jsx]: fixed

File: frontend/src/container/Header.jsx (hunk 2 above)
Comment (Skylsmoi) [reply frontend/src/container/Header.jsx]: fixed

File: frontend/src/container/Header.jsx (hunk 1 above)
Comment (Skylsmoi) [reply frontend/src/container/Header.jsx]: fixed

**Pull request merged**